packages/web-react/src/components/Tag/Tag.stories.tsx

```diff
@@ -20,14 +21,16 @@ export default {
       control: 'text',
     },
   },
-};
+} as ComponentMeta<typeof Tag>;
 
-const Template = (args: TagProps) => <Tag {...args} />;
+const Template = (args: typeof Tag) => <Tag {...args} />;
 
 export const ExampleTag = Template.bind({});
 ExampleTag.args = {
   color: 'default',
   size: 'medium',
   theme: 'dark',
   children: 'Tag example',
+  UNSAFE_style: { opacity: 1 },
```

Comment:
I think that these props should not be here because it is not necessary. Look at for example Alert story. Storybook itself reads the props from types and therefore it is not necessary to write and example for all of them here.

Reply:
Otherwise it looks good to me.

Reply:
Ok, i  removed these unnecessary properties